Bexchange.Domain/Enum/Roles.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BexchangeAPI.Domain.Enum
+{
+    public enum Roles
+    {
+        User,
```

Comment:
do not use consecutively. Use geometric progression (1, 2, 4, 8...). It can help in the future